### DISCUSSION TITLE
How to work in water/hydrological year rather than calendar year to compute indices?

### DISCUSSION THREAD

**Discussion by ArcticSnow** [Ouranosinc/xclim#1588] [Questions] [answered]
### Setup Information

- Xclim version: 0.41.0
- Python version: 3.9
- Operating System: Ubuntu 20.04

Thank you for the great library!

It might be a simple thing, bu I cannot figure out how to compute a number of the atmospheric indices (e.g. freezing degree day) for water years rather than calendar year? My focuses on winters and such options would very useful.

Thank you.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by aulemahal** [comment, marked as answer]
Hi @ArcticSnow (cool name btw)

When it comes to "resampling frequencies", xclim follows xarray which itself follows a subset of pandas. This page of the doc shows a bunch of examples : https://docs.xarray.dev/en/stable/generated/xarray.cftime_range.html

Thus, a "year" is always 12 months, but you can choose the month it starts in. For example, for computing the freezing degree days accumulation from the 1st of October to the 31st of September, one would do:
```python3
import xclim as xc

# ... data stuff ...

out = xc.atmos.freezing_degree_days(tas=ds.tas, freq='AS-OCT')
```
Here the result has the accumulation from (say) "2022-10-01" to "2023-09-31" assigned to the date "2022-10-01".

Does this fill your need ?

**Comment by ArcticSnow** [comment]
Awesome! That was it! 
Thank you @aulemahal  :+1:
